core/deployment/src/main/java/io/quarkus/deployment/configuration/ConfigDefinition.java

```diff
@@ -58,6 +59,10 @@
 
     public static final String NO_CONTAINING_NAME = "<<ignored>>";
 
+    private static final String QUARKUS_NAMESPACE = "quarkus";
+    private static final List<String> FALSE_POSITIVE_QUARKUS_CONFIG_MISSES = Arrays
+            .asList(QUARKUS_NAMESPACE + ".live-reload", QUARKUS_NAMESPACE + ".debug");
```

Comment:
It's minor but shouldn't we add the `.` at the end of the strings?

That would be safer IMHO.

Reply:
Makes sense! Fixed :)